client/jetpack-cloud/sections/partner-portal/license-list/index.tsx

```diff
@@ -12,9 +12,25 @@ import CardHeading from 'calypso/components/card-heading';
 import DocumentHead from 'calypso/components/data/document-head';
 import LicenseListItem from 'calypso/jetpack-cloud/sections/partner-portal/license-list-item';
 import LicensePreview from 'calypso/jetpack-cloud/sections/partner-portal/license-preview';
+import LicenseStateFilter from 'calypso/jetpack-cloud/sections/partner-portal/license-state-filter';
 
-export default function LicenseList() {
+export enum LicenseStates {
+	All = 'all',
+	Detached = 'detached',
+	Attached = 'attached',
+	Revoked = 'revoked',
+}
+
+interface Props {
+	licenseState?: string;
+	search?: string;
+}
+
+export default function LicenseList( { licenseState, search }: Props ): ReactElement {
```

Comment:
We should add defaults for optional params:
```suggestion
export default function LicenseList( { licenseState = LicenseStates.All, search = '' }: Props ): ReactElement {
```